lib/FieldOrganization/FieldOrganization.js

```diff
@@ -83,8 +85,9 @@ const FieldOrganization = ({
     // eslint-disable-next-line react-hooks/exhaustive-deps
     [selectedOrganization, disabled],
   );
+  const label = <FormattedMessage id={labelId} />;
 
-  return (
+  return isNonInteractive ? <OrganizationValue label={label} value={selectedOrganization?.name} /> : (
```

Comment:
it's a field component, but can render only value?

Reply:
Yes, if it's in isNonInteractive mode.